services/shhext/chat/x3dh.go

```diff
@@ -86,8 +88,12 @@ func ExtractIdentity(bundle *Bundle) (string, error) {
 		return "", err
 	}
 
+	var signatureMaterial []byte
+	for installationID, signedPreKey := range bundle.GetSignedPreKeys() {
+		signatureMaterial = append([]byte(installationID), signedPreKey.SignedPreKey...)
```

Comment:
@cammellos what is the intent inside this loop? It looks like the calculated `signatureMaterial` is getting thrown away until the last value of `bundle.GetSignedPreKeys()`